<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Git::Internal::RemoteAdditionDialog</class>
 <widget class="QDialog" name="Git::Internal::RemoteAdditionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>93</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Remote</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="Utils::FancyLineEdit" name="nameEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="urlLabel">
     <property name="text">
      <string>URL:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="Utils::FancyLineEdit" name="urlEdit"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::FancyLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">utils/fancylineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Git::Internal::RemoteAdditionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>283</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Git::Internal::RemoteAdditionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
